### ISSUE TITLE
Allow changing background color of the header controls 

### ISSUE THREAD

**Issue by samuel27m** [stephy/CalendarPicker#260]
I couldn't find any way of setting the background color of the header controls, and can't see anything on documentation.
![image](https://user-images.githubusercontent.com/20863811/103027378-a2240500-454d-11eb-921a-cce671f03ede.png)

Is this a feature that needs to be implemented?

**Comment by helloWorld2020102** [comment]
you could use the `monthYearHeaderWrapperStyle={{backgroundColor: 'red'}}`
but this allows u to only give styles to the 'December 2020' portion

for those previous and next portions, I used
```
previousComponent={this.renderPreviouIcon()}
nextComponent={this.renderNextIcon()}
```
For the complete styling of the header, I didn't find any prop... Please let me know if you do...

**Comment by samuel27m** [comment]
@helloWorld2020102 Thanks for the suggestion.
I tried that before already but no luck. Would be nice if we could just style the whole header. I was trying to achieve something like this: 

![image](https://user-images.githubusercontent.com/20863811/103035959-71010000-4560-11eb-8379-d3c6d02515af.png)

But if there is currently no option for it I guess I can live with it, but would be nice if there was an option for the whole header - e.g. if I want to give `margin` between calendar and calendar header

**Comment by peacechen** [comment]
We will need a new prop for that (e.g. `headerControlsStyle`). Please submit a PR if you have time to add it.

**Comment by peacechen** [comment]
Published in 7.0.7
Happy New Year 🎆

**Comment by samuel27m** [comment]
Thanks @peacechen :)
